### PR TITLE
Transition feature

### PR DESCRIPTION
**Pull request by nvescio**
Summary
 - Transition added to all technologies

Screenshots
Chrome
transition-1
transition-2

Trello Card
https://trello.com/c/N9uksGkU/25-css-transitions

### CODE REVIEW

File: training_maquetado/scss/tech.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
     box-sizing: border-box;
 }
 
-.tech-container:before {
+.tech-container::before {
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss]: use & inside class, you also repeated the  :before

File: training_maquetado/scss/tech.scss [hunk 2]
```diff
@@ -25,6 +25,12 @@
     top: -10px;
     width: 100%;
     z-index: -1;
+    transition: all 0.3s ease;
+}
+
+.tech-container:hover::before {
```
Comment (CarlosHWolox) [inline training_maquetado/scss/tech.scss]: same as above

File: training_maquetado/scss/tech.scss [hunk 3]
```diff
@@ -1,60 +1,55 @@
-.tech-container {
+.tech-grid {
     display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 0 350px 80px;
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: I think this is not needed

File: training_maquetado/scss/tech.scss [hunk 4]
```diff
@@ -1,60 +1,55 @@
...
-    height: 100%;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    width: 100%;
-    z-index: -1;
-}
-
-.tech-icon {
+      &:hover::before {
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: check indentation

File: training_maquetado/scss/tech.scss [hunk 5]
```diff
@@ -1,60 +1,55 @@
...
-    align-content: space-between;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    margin: 0 auto 80px;
-    width: 50%;
-}
+  }
+
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: remove spaces

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: training_maquetado/scss/tech.scss [hunk 6]
```diff
@@ -2,9 +2,10 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
-  margin: 0 350px 80px;
+  margin: 0 auto;
+  width: 1000px;
 }
-
+
```
Comment (FrankIglesias) [inline training_maquetado/scss/tech.scss]: Delete space

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**